apps/playground/src/app/components/lists/repeater/nested-repeater.component.ts

```diff
@@ -28,9 +28,11 @@ type ListItem = {
         position: absolute;
         top: 0;
         left: 140px;
+        /* spell-checker:disable */
```

Comment:
if this is for the color names, why not add these values to the spell check ignore list. esp since its css colors